aten/src/ATen/TensorIterator.h

```diff
@@ -250,7 +250,6 @@ struct TORCH_API TensorIteratorBase : public impl::MetaBase {
   using PtrVector = SmallVector<char*, 4>;
   using StrideVector = SmallVector<int64_t, 6>;
 
-  TensorIteratorBase();
```

Comment:
This is still default constructable, right?

Reply:
Right, but it's not need to write explicitly.